### DISCUSSION TITLE
请问shizuku支持使用uid1000运行吗

### DISCUSSION THREAD

**Discussion by RimuruChan** [RikkaApps/Shizuku#241] [Q&A] [answered]
如果在uid1000启动shizuku server，是不是能直接调用devicepolicymanager之类的服务了呢

**Comment by RikkaW** [comment, marked as answer]
1. root 启动的 Shizuku 或者使用 Sui
2. 在 UserService 中 Os.setuid 或 JNI 中 setuid

**Reply by RimuruChan** [reply]
非常感谢！
